modules/openfast-library/src/FAST_Lin.f90

```diff
@@ -5346,6 +5351,12 @@ SUBROUTINE FAST_CalcSteady( n_t_global, t_global, p_FAST, y_FAST, m_FAST, ED, BD
                m_FAST%Lin%AzimIndx = 1
                m_FAST%Lin%CopyOP_CtrlCode = MESH_UPDATECOPY
             end if
+            ! Forcing linearization if time is close to tmax
+            if ((t_global> p_FAST%TMax - 1.5*TwoPi_D/ED%y%RotSpeed) .and. .not.m_FAST%Lin%FoundSteady)  then
+               call WrScr('')
+               call WrScr('[WARNING] Steady state not found before end of simulation. Forcing linearization')
+               m_FAST%Lin%ForceLin = .True. 
```

Comment:
Should you also warn that the linearized model may not be sufficiently representative of the solution in steady state?

Reply:
I've adding a final warning at the end.